src/i18n/locales/en-US.json

```diff
@@ -78,18 +78,26 @@
   "components.maintenancemodal.explanation": "Yoroi is in maintenance mode. You can still access your funds through any other wallet software.",
   "components.maintenancemodal.learnMore": "Learn more",
   "components.maintenancemodal.title": "Temporary Maintenance",
-  "components.receive.addressmodal.BIP32path": "BIP32 path:",
+  "components.receive.addressmodal.BIP32path": "Derivation path",
   "components.receive.addressmodal.copiedLabel": "Copied",
   "components.receive.addressmodal.copyLabel": "Copy address",
-  "components.receive.addressmodal.walletAddress": "Your wallet address",
+  "components.receive.addressmodal.spendingKeyHash": "Spending key hash",
+  "components.receive.addressmodal.stakingKeyHash": "Staking key hash",
+  "components.receive.addressmodal.title": "Verify address",
+  "components.receive.addressmodal.walletAddress": "Address",
   "components.receive.addressverifymodal.afterConfirm": "Once you tap on confirm, validate the address on your Ledger device, making sure both the path and the address match what is shown below:",
   "components.receive.addressverifymodal.title": "Verify Address on Ledger",
+  "components.receive.addressview.copiedAddressLabel": "Copied",
+  "components.receive.addressview.copyAddressLabel": "Copy address",
```

Comment:
It might be a good idea to reuse strings that are already there, as any new key must be translated and we rely on volunteers from the community. In this case for instance you could just use `components.receive.addressmodal.copyLabel`

Reply:
Roger that, gonna be moving the most frequent texts to global later.